components/centraldashboard/public/components/main-page.js

```diff
@@ -173,11 +173,14 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
      * @param {Event} ev AJAX-response
```

Comment:
Please make sure the tests [***here***](https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/public/components/main-page_test.js) are updated and pass

Reply:
Done.

Reply:
Done.